src/Features/Core/Portable/EditAndContinue/DebuggingSession.cs

```diff
@@ -98,19 +98,21 @@ internal sealed class DebuggingSession : IDisposable
         internal readonly IManagedEditAndContinueDebuggerService DebuggerService;
 
         private readonly DebuggingSessionTelemetry _telemetry;
-
+        private readonly ManagedEditAndContinueCapabilities _capabilities;
```

Comment:
>private readonly ManagedEditAndContinueCapabilities _capabilities; [](http://example.com/codeflow?start=8&length=66)

Nit: This can just be 
```
internal readonly ManagedEditAndContinueCapabilities Capabilities;
```